src/style.css

```diff
@@ -1,3 +1,53 @@
 body {
   margin: 0;
+  font-family: 'Roboto', sans-serif;
+  background-color: lightgray;
+}
+
+header {
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  padding: 5px 50px;
+  background-color: rgba(255, 255, 255, 1);
+}
+
+ul {
+  display: flex;
+  list-style: none;
```

Comment:
Don't use tag names for styling (except 'html' and 'body'). You should style elements using classes instead of tag names to avoid styling conflicts and improve the reusability of your styles. Consider adding a class to the 'ul' element and styling that class.